<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDoc_New"/>
   <addaction name="actDoc_Open"/>
   <addaction name="actCloseAll"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="actFont"/>
   <addaction name="separator"/>
   <addaction name="actViewMode"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actDoc_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/new.PNG</normaloff>:/image/image/new.PNG</iconset>
   </property>
   <property name="text">
    <string>新建文档</string>
   </property>
   <property name="toolTip">
    <string>新建文档</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/open.PNG</normaloff>:/image/image/open.PNG</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/cut.PNG</normaloff>:/image/image/cut.PNG</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/copy.PNG</normaloff>:/image/image/copy.PNG</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/paste.PNG</normaloff>:/image/image/paste.PNG</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/actFont.PNG</normaloff>:/image/image/actFont.PNG</iconset>
   </property>
   <property name="text">
    <string>设置格式</string>
   </property>
   <property name="toolTip">
    <string>设置格式</string>
   </property>
  </action>
  <action name="actCloseAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/closeAll.PNG</normaloff>:/image/image/closeAll.PNG</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
   <property name="toolTip">
    <string>关闭全部子窗口</string>
   </property>
  </action>
  <action name="actTile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/actTile.PNG</normaloff>:/image/image/actTile.PNG</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
   <property name="toolTip">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/actCascade.PNG</normaloff>:/image/image/actCascade.PNG</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
   <property name="toolTip">
    <string>级联展开</string>
   </property>
  </action>
  <action name="actViewMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/actMDIViewMode.PNG</normaloff>:/image/image/actMDIViewMode.PNG</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
   <property name="toolTip">
    <string>MDI显示模式</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/image/actExit.PNG</normaloff>:/image/image/actExit.PNG</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
